.python-version

```diff
@@ -0,0 +1 @@
+2.7.5
```

Comment:
is this needed?  i never had this problem.

Reply:
i remember you mentioning it before, just wonder if it is needed for anyone else - fine if it is harmless, but wonder if that is the case (eg will it restrict a person using python 2.7.4?)

Reply:
I'm getting this in all of my node projects. It's what I get for having Python 3 set as my default. There's a dependency that blows up unless you're on `2.7.x`.

Reply:
I can gitignore it, I suppose. Just wondering it it could save someone else some grief.

Reply:
yeah, fine to leave it i think.  i don't know if it'll cause grief.  fwiw, i've found it easiest to keep the default OS python and pyenv/virtualenv for a peaceful pain free life.  (and rvm/npm etc)  changing default python on ubuntu breaks many things, or so i seem to recall.